src/CMakeLists.txt

```diff
@@ -35,12 +35,27 @@ configure_file("celeritas_config.h.in" "celeritas_config.h" @ONLY)
 
 #----------------------------------------------------------------------------#
 
-# TODO: add build flags
-set(CELERITAS_CMAKE_STRINGS
-    "static const char celeritas_rng[] = \"${CELERITAS_RNG}\";\n"
-)
-set(BUILD_TYPE ${CMAKE_BUILD_TYPE})
-foreach(_var BUILD_TYPE CLHEP_VERSION Geant4_VERSION VecGeom_VERSION)
+# Get host system information
+if(NOT DEFINED CELERITAS_HOSTNAME)
+  set(_hostname "$ENV{LMOD_SYSTEM_NAME}")
+  if(NOT _hostname)
+    cmake_host_system_information(RESULT _hostname QUERY HOSTNAME)
```

Comment:
I am not sure if it is the case here, but often fetch the hostname requires network access.  So related question, do we require network access to build `Celeritas`?

Reply:
(here there is obviously a work-around but if we do support non-network-access build we should document the work-around)

Reply:
I was expecting it to use $HOSTNAME or other system utilities, because we definitely don't want to hit the network. (Remember when VecGeom validated all the XML input and required 15 seconds of network activity to run each test? Good times.)

I just checked the cmake source code and this simply calls `uname` on unixy platforms and an equivalent on windows. So no network access.

Reply:
Indeed, it seems that uname just read the content of `/proc/sys/kernel/hostname` or something like.  So fair enough.